recipes-graphics/l4t-graphics-demos/l4t-graphics-demos.inc

```diff
@@ -2,7 +2,7 @@ HOMEPAGE = "https://developer.nvidia.com/embedded"
 
 COMPATIBLE_MACHINE = "(tegra)"
 
-L4T_DEB_SOCNAME = "t194"
+L4T_DEB_SOCNAME ?= "t194"
```

Comment:
No, this should be straight assignment, so we're always fetching the `t194` version.  The content doesn't actually vary between the two, so we don't need to account for the SoC name.